nextcloudappstore/core/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{% static 'assets/img/favicon.png' %}"/>
     <title>Nextcloud App Store{% block head-title %}{% endblock %}</title>
```

Comment:
I'd also go for [subpage] - App store - Nextcloud. I hate pages where the important part of the title is last. That makes finding the right tab a real pain. :)

Other than that everthing looks fine for me.